pages/docs/end-user-guide/nominating.mdx

```diff
@@ -0,0 +1,129 @@
+Let's explore the significance of these icons and the legend to gain a deeper understanding of the information they convey.
+
+<img src="/img/nominating/stake-page-legend.png" width="100%" height="100%" />
+
+- **Next Session**: The "next session" is showing at that time the validator will be elected in the next session.
+
+- **Produced Blocks**: Display the quantity of blocks created by the validator for that epoch, is reset at teh start of every epoch.
+
+- **Online Message**: An online message indicates that a validator is online and an active check done by the network to determine if the validator is online. This icon is seen more where there is more valiators per epoch than the blocks procued per epoch. Once they produce a block this icon will be replaced with produced block icon.
+
+- **Nominating**: Should you have already nominated and have that wallet open, will display this ico next to the validators you have nominated.
+
+- **Oversubscribed**: The "Oversubscribed" status appears when a validator accumulates over 256 active nominations. Although a validator may have numerous nominators shared with other validators, Avail consensus ensures an optimal distribution of nominators during elections to maximize AVL staking across the network. However, there are cases where a validator exceeds the 256 active nominations limit. In such instances, only the top 256 wallets will receive rewards, while wallets in the red section will not be eligible for any rewards.
+
+- **Slashed**: Validators can be slashed for various reasons, including downtime, misbehavior, or other protocol violations. Slashing involves a penalty, which can result in a loss of part or all all supporters (includig yours) stake.
+
+- **Block Nominations**: Block nominations indicates the validator is not looking for more nominations and has therefore blocked any community nominations.
+
+- **Era Points**: Era points are a measure of a validator's contribution to the network over a specific period (era). Validators earn era points based on their performance, including producing and validating blocks, and these points influence their overall reputation and rewards. 
+
+
+### Nominating VS Nomination Pools
+Nominating and nomination pools are both integral parts of the staking process in blockchain networks like Avail. However, they serve slightly different purposes and involve distinct mechanisms:
+
+- **Nominating**: Nominating refers to the process of selecting validators to delegate one's stake to in order to participate in the network's consensus mechanism and earn rewards. When users nominate validators, they essentially entrust them with their tokens to contribute to the security and operation of the network. By nominating validators that perform well and uphold the network's integrity, users can earn staking rewards in return for their contribution.
```

Comment:
I'm pretty sure there's a minimum number of tokens a user needs to have to be an independent nominator.

Can we add that number here?

Reply:
Hi

Apologies on the delay, been crazy my side.

There is, but to be honest it keeps changing in each testnet. On kate testnet was x, goldberg y, turning z. Might be something we add when we know final config for mainnet